lib/src/models.dart

```diff
@@ -34,6 +34,7 @@ class DownloadTaskStatus {
 /// * [progress] the latest progress value of a download task
 /// * [url] the download link
 /// * [filename] the local file name of a downloaded file
+/// * [additionalInfo] Additional information of a downloaded file
```

Comment:
Maybe change to something like `Optional, extra string associated with this task`